### PR TITLE
TPO-197: Add jscs.

### PR DESCRIPTION
**Pull request by ziemek**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -68,15 +68,15 @@ module.exports = function(config) {
     logLevel: config.LOG_INFO,
 
     junitReporter : {
-      outputDir: 'reports/',
+      outputDir: 'reports/junit/junit/',
```
Comment (skwoka) [inline karma.conf.js]: In other projects, junit reports are stored in `reports/junit/`. Just wondering why reports are stored two levels deep here.

File: Gruntfile.js [hunk 2]
```diff
@@ -42,10 +50,28 @@ module.exports = function (grunt) {
...
+          }
+        }
+      }
     }
   });
 
   grunt.registerTask('default', ['build']);
-  grunt.registerTask('build', ['newer:jshint', 'karma', 'concat']);
-  grunt.registerTask('test', ['newer:jshint', 'karma']);
+  grunt.registerTask('build', ['newer:jshint', 'jscs', 'wiredep', 'karma', 'concat']);
```
Comment (skwoka) [inline Gruntfile.js]: I noticed in your other PRs, `build` task calls `test`. How about changing this to `['test', 'concat']`?

**Comment by skwoka** [comment]
Finished my review. Back over to you @ziemek.

File: karma.conf.js (hunk 1 above)
Comment (ziemek) [inline karma.conf.js]: whoops, was on auto pilot 👍

File: Gruntfile.js (hunk 2 above)
Comment (ziemek) [inline Gruntfile.js]: 👍 good call

**Comment by ziemek** [comment]
Fixed, ready for another review.

**Comment by skwoka** [comment]
🐙

**Pull request merged**